packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx

```diff
@@ -62,6 +62,7 @@ export const ArrowBlockButton = React.forwardRef<HTMLButtonElement, ArrowBlockBu
           justifyContent: 'center',
           borderColor: theme.colors.$neutralAlpha100,
           alignItems: 'center',
+          padding: `${theme.space.$1x5} ${theme.space.$3} ${theme.space.$1x5} ${theme.space.$2x5}`,
```

Comment:
Why is this necessary? Isn't it going to make most screens deviate from designs now?

Reply:
This is the padding that the designs have.